tests/drivers/counter/counter_basic_api/src/test_counter.c

```diff
@@ -276,7 +276,7 @@ static void alarm_handler(struct device *dev, u8_t chan_id, u32_t counter,
 			(counter + top - now) : (counter - now);
 	}
 
-	zassert_true(diff < counter_us_to_ticks(dev, processing_limit_us),
+	zassert_true(diff <= counter_us_to_ticks(dev, processing_limit_us),
```

Comment:
so i wonder how does it help? assert checks if counter handler was processed within given time from which hardware event occured. Limit was set to arbitrary value 1ms as it was expected that it should be more than enough for the processor to handle that event. If it is not enough then limit should be increased or maybe it actually shows some issue with the specific counter implementation.

Reply:
Yes equal help because with 1Hz, the convertion `counter_us_to_ticks(dev, processing_limit_us)` give 0 ticks and `diff  `is also equal to 0.